.gitignore

```diff
@@ -6,3 +6,5 @@ _build/
 .vscode/
 node_modules/
 package-lock.json
+*.log
+.pytest*/
```

Comment:
You're probably going to add quite a few things to this piece by piece. You might consider starting with one [off-the-shelf](https://github.com/github/gitignore).

Reply:
I think this can be done in another PR.

Reply:
Opened #14